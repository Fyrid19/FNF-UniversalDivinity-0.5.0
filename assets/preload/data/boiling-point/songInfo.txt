Rareblin
VS Impostor Team
0xFF8A00A2