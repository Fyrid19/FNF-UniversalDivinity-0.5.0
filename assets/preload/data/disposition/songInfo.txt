Composer: Shredboi (Cover by FyriDev)