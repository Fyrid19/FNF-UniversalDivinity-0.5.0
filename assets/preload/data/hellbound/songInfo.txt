Aura
AVeryCringeUser, Aura
0xFFFFFFFF