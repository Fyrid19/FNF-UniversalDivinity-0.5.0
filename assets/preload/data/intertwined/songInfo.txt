TheBanana
Aura
0xFF1D3FFF