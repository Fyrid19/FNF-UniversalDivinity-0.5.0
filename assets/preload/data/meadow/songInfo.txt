Ayo
-Jay-
0xFF00B515