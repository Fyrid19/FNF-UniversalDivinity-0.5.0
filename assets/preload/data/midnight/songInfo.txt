FyriDev
FyriDev
0xFF1D3FFF