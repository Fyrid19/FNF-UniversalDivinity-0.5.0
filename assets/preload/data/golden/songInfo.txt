BoxHaze
-Jay-, Aura
0xFFFFD146